### PR TITLE
added canvas assignment name strings to config

### PR DESCRIPTION
**Pull request by mbogor**
#### What's this PR do?
adds canvas assignment name string to config and calls attendance and graduation functions upon the start of the app
#### Where should the reviewer start?
config, attendancesync, and pollCanvasForGrads
#### How should this be manually tested?
run in node
#### What are the relevant tickets?
https://admin.crisistextline.org/jira/browse/INT-251

### CODE REVIEW

File: jobs/scheduling/AttendanceSync.js [hunk 1]
```diff
@@ -10,7 +10,8 @@ var fs = require('fs');
 new CronJob(CONFIG.time_interval.gtw_attendance_sync_with_canvas, function () {
    startJobByQueryingForGTWSessions();
 }, null, true);
```
Comment (apackin) [inline jobs/scheduling/AttendanceSync.js]: This Boolean isn't working to start the function on run?

**Comment by apackin** [comment]
👍

File: jobs/scheduling/AttendanceSync.js (hunk 1 above)
Comment (mbogor) [inline jobs/scheduling/AttendanceSync.js]: nope, it starts the job automatically which means it doesn't ever exit the CronJob and just waits for the specified time interval indefinitely. if we didn't have the 'true' there, we would have to set the cronjob equal to a variable (let's say 'job') and then initiate it with job.start();

**Comment by tongxiang** [comment]
👍

**Comment by tongxiang** [comment]
Actually--getting an error on test. Is this something we're concerned about? 

0 GMT-0400 (EDT)] Graduating from assignment: 1185
Unhandled rejection TypeError: Cannot read property 'id' of undefined
    at /Users/tongxiang/ctl/Elmer/jobs/scheduling/AttendanceSync.js:170:51
    at tryCatcher (/Users/tongxiang/ctl/Elmer/node_modules/bluebird/js/release/util.js:16:23)
    at Promise._settlePromiseFromHandler (/Users/tongxiang/ctl/Elmer/node_modules/bluebird/js/release/promise.js:504:31)
    at Promise._settlePromise (/Users/tongxiang/ctl/Elmer/node_modules/bluebird/js/release/promise.js:561:18)
    at Promise._settlePromise0 (/Users/tongxiang/ctl/Elmer/node_modules/bluebird/js/release/promise.js:606:10)
    at Promise._settlePromises (/Users/tongxiang/ctl/Elmer/node_modules/bluebird/js/release/promise.js:685:18)
    at Async._drainQueue (/Users/tongxiang/ctl/Elmer/node_modules/bluebird/js/release/async.js:138:16)
    at Async._drainQueues (/Users/tongxiang/ctl/Elmer/node_modules/bluebird/js/release/async.js:148:10)
    at Immediate.Async.drainQueues [as _onImmediate] (/Users/tongxiang/ctl/Elmer/node_modules/bluebird/js/release/async.js:17:14)
    at tryOnImmediate (timers.js:543:15)
    at processImmediate [as _immediateCallback] (timers.js:523:5)

**Comment by tongxiang** [comment]
Most likely not caused by this PR, though.

**Comment by mbogor** [comment]
@tongxiang the assignment name was changed from 'Attend 1 Observation' to 'Attend An Observation' in Canvas which was causing errors on our side. i've changed our script to reflect the change

**Pull request merged**